index.html

```diff
@@ -35,5 +35,127 @@ <h1>I’m Benson.<br>Glad to see you!</h1>
             </ul>
         </section> 
     </header>
```

Comment:
[Optional] The headline section of your project lacks some background images which made it look a bit scanty, please you could try to see if you can add them as well, although its not part of this milestone but it would be nice if you add them 😄

Reply:
- Being optional will enforce later on after consultations as trying to add but distorting whole design